### PR TITLE
Feature/iclr workshop

### PR DESCRIPTION
**Pull request by mspector**
This is the basic infrastructure for starting the ICLR 2018 Workshop track.

I also added a webfield file for the ICLR 2018 landing page (the one that links to both the Conference and the Workshop), since I couldn't actually find the webfield file anywhere. Did I just miss it?

The settings for this Workshop are identical to the conference, except that there are no area chairs, and paper submissions are non-anonymous.

EDIT: For what it's worth: I have been testing this pretty thoroughly, and am confident that everything is working. The submission process function is the "fast" version, which does not generate any comment invitations, but the invitations.py script can create them after submissions have been made.

### CODE REVIEW

**Comment by pmandler** [comment]
The duedates are already passed in several places. 
config.py line 312 official review
                 line 384 meta review
                 line 515 add bids
It might be left on purpose while you're waiting for the real due dates.
In config.py on line 424 it refers to "area chairs".
In webfield_tabs.js line 10 it needs a '/' before "ProgramChairs".

**Comment by mspector** [comment]
Thanks @pmandler . Just so you know, you can make comments inline by clicking on the plus sign next to the line itself, when you're reviewing the code.

File: venues/ICLR.cc/2018/Workshop/process/submissionProcess.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      members: note.content.authorids.concat(note.signatures),
+      readers: [CONF, 'everyone'],
+      signatories: [authorGroupId]
+    };
+    return or3client.or3request(or3client.grpUrl, authorGroup, 'POST', token);
+  })
+  .then(result => {
+    return or3client.or3request(or3client.inviteUrl, addRevisionInvitation, 'POST', token);
+  })
+  // .then(result => {
```
Comment (melisabok) [inline venues/ICLR.cc/2018/Workshop/process/submissionProcess.js]: Please remove commented code

**Comment by melisabok** [comment]
I couldn't take a look in detail but can we merge this?

**Comment by mspector** [comment]
Yes, I think this can be merged. I'm waiting on some due dates for various invitations but I addressed the other comments that Pam made.

**Pull request merged**